mam/api/BUILD

```diff
@@ -16,6 +17,15 @@ cc_library(
     ],
 )
 
+# Build command:
+# bazel build --spawn_strategy=local --define=workspace=$(bazel info workspace)  --define=output_base=$(bazel info output_base) --crosstool_top='@iota_toolchains//tools/emscripten:toolchain' --cpu=emscripten --host_crosstool_top='@bazel_tools//tools/cpp:toolchain' //mam/api:api.js
+
+emcc_binary(
+    name = "api.js",
+    linkopts = ["-s LINKABLE=1 -s EXPORT_ALL=1"],
```

Comment:
Does that mean we are exporting absolutely every function ?

Reply:
yes, every non static function, we can alternatively specify the functions we want exported, no middle solution